clients/admin-ui/src/features/privacy-requests/configuration/TwilioEmailConfiguration.tsx

```diff
@@ -45,7 +45,10 @@ const TwilioEmailConfiguration = () => {
     api_key: string;
   }) => {
     const result = await createMessagingConfigurationSecrets({
-      twilio_api_key: value.api_key,
+      details: {
```

Comment:
I would have suggested a BaseConfiguration component given these components are identical except for the data params.